### DISCUSSION TITLE
Should we release 0.70.2?

### DISCUSSION THREAD

**Discussion by dmytrorykun** [reactwg/react-native-releases#34] [Patches]
## Released - please refer to #36 for followups

Current Release: [0.70.1](https://github.com/facebook/react-native/releases/tag/v0.70.1)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Issues we are aware of

1. https://github.com/facebook/react-native/issues/34656
2. https://github.com/facebook/react-native/issues/34673
3. https://github.com/facebook/react-native/issues/34665

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/be7c50fefd7f13201fb538ded93d91b374341173
2. https://github.com/facebook/react-native/commit/3d7e1380b4e609f5340ee80c19d566b17e620427
4. https://github.com/facebook/react-native/commit/f1645560376b734a87f0eba1fef69f6cba312cc1

#### Local commits to backport to main

1.

**Comment by renchap** [comment]
`Animated` has a broken behaviour on 0.70: https://github.com/facebook/react-native/issues/34665
It breaks the drawer implementation in `react-native-gesture-handler` for example and is a blocker for upgrading to 0.70 in multiple apps.

**Comment by gabrieldonadel** [comment]
I just noticed that we missed https://github.com/facebook/react-native/commit/be7c50fefd7f13201fb538ded93d91b374341173 when we cherry-picked https://github.com/facebook/react-native/commit/4b9382c250261aab89b271618f8b68083ba01785 on 0.70.1 which is required otherwise https://github.com/facebook/react-native/blob/03cb5aca3f5b8594287608265d4bde7104f07b87/Libraries/Components/Keyboard/KeyboardAvoidingView.js#L88 throws an `undefined is not a function` error when "Prefer Cross-Fade Transitions" is enabled  

![image](https://user-images.githubusercontent.com/11707729/190491251-2c678d8c-3081-4585-8579-e6d300cab907.png)

**Reply by fungilation** [reply]
Reproducing same. On iOS 15.7, iPhone 12

**Comment by billnbell** [comment]
Getting weird error with metro

https://github.com/facebook/react-native/issues/34714

**Reply by cortinico** [reply]
This is not the thread to report issues but to ask for pick requests, for fixes that landed on master.

**Comment by billnbell** [comment]
then lets add https://github.com/facebook/react-native/commit/be7c50fefd7f13201fb538ded93d91b374341173

**Reply by tido64** [reply]
~~Sorry, this thread is not for feature requests. New features can break things, and we don't want to risk that in patch releases. Please see [What is a qualified pick request?](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) from the description in OP.~~ Sorry, it looks like we do need it because we cherry-picked something else that broke it here: https://github.com/reactwg/react-native-releases/discussions/34#discussioncomment-3656859

**Comment by kelset** [comment]
we need to realign the deps: https://github.com/facebook/react-native/issues/34714 (to latest 0.72.3)

**Reply by kelset** [reply]
also bump CLI

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/34803

**Reply by kelset** [reply]
commit: https://github.com/facebook/react-native/commit/f1645560376b734a87f0eba1fef69f6cba312cc1

**Comment by Kudo** [comment]
hope we could also pick this https://github.com/facebook/react-native/commit/3d7e1380b4e609f5340ee80c19d566b17e620427

**Comment by tido64** [comment]
It would be great if we can pick the AGP bump as it resolves build ordering issues: https://github.com/facebook/react-native/commit/9f6711fda084d95d422a3ec9aed82bce87459463

cc @cortinico

**Reply by cortinico** [reply]
As suggested by @kelset is maybe better to don't bump AGP versions in point releases. As this fixes just an annoyance, I think it's probably fine to don't bump it.

**Comment by brunoducattii** [comment]
Sorry if i bothering, but it is ok to launch 0.69.6 instead 0.70.2? I don't know if i should use 0.69.6 or 0.70.1.

Thank you in advance.

**Reply by evan1715** [reply]
yeah, you can use 0.69.6 instead of trying to use 0.70.1 or 0.70.2. if you're having hermes compiling issues on iOS from 0.70.1 with xcode, like i am waiting for 0.70.2 to fix it before releasing the app, then 0.69.6 is probably best. if you're not facing any hermes issues, then the latest version of anything is typically the best.

**Reply by cortinico** [reply]
Sorry but this not the correct please to ask questions about which version to use, as that is probably specific to your environment. Please consider posting this on Stackoverflow.
